src/utils/slpReader.ts

```diff
@@ -305,26 +305,63 @@ export function parseMessage(command: Command, payload: Uint8Array): EventPayloa
           playerIndex,
           port: playerIndex + 1,
           characterId: readUint8(view, 0x65 + offset),
-          characterColor: readUint8(view, 0x68 + offset),
-          startStocks: readUint8(view, 0x67 + offset),
           type: readUint8(view, 0x66 + offset),
+          startStocks: readUint8(view, 0x67 + offset),
+          characterColor: readUint8(view, 0x68 + offset),
+          teamShade: readUint8(view, 0x6c + offset),
+          handicap: readUint8(view, 0x6d + offset),
           teamId: readUint8(view, 0x6e + offset),
-          controllerFix,
-          nametag,
-          displayName,
-          connectCode,
-          userId,
+          staminaMode: !!readUint8(view, 0x6c + playerIndex * 0x24, 0x01),
+          silentCharacter: !!readUint8(view, 0x6c + playerIndex * 0x24, 0x02),
+          lowGravity: !!readUint8(view, 0x6c + playerIndex * 0x24, 0x04),
+          invisible: !!readUint8(view, 0x6c + playerIndex * 0x24, 0x08),
+          blackStockIcon: !!readUint8(view, 0x6c + playerIndex * 0x24, 0x10),
+          metal: !!readUint8(view, 0x6c + playerIndex * 0x24, 0x20),
```

Comment:
I prefer wrapping things in `Boolean(value)` instead of `!!value` for boolean coersion since I think it's clearer.